eng/common/scripts/ChangeLog-Operations.ps1

```diff
@@ -40,28 +40,45 @@ function Get-ChangeLogEntriesFromContent {
   $changeLogEntries = [Ordered]@{}
   try {
     # walk the document, finding where the version specifiers are and creating lists
-    $changeLogEntry = $null
     foreach ($line in $changeLogContent) {
       if ($line -match $RELEASE_TITLE_REGEX) {
         $changeLogEntry = [pscustomobject]@{ 
           ReleaseVersion = $matches["version"]
           ReleaseStatus  =  $matches["releaseStatus"]
           ReleaseTitle   = "## {0} {1}" -f $matches["version"], $matches["releaseStatus"]
           ReleaseContent = @()
+          FeaturesAdded = @()
+          BreakingChanges = @()
+          KeyBugsFixed = @()
+          Fixed = @()
         }
         $changeLogEntries[$changeLogEntry.ReleaseVersion] = $changeLogEntry
       }
       else {
         if ($changeLogEntry) {
+          if ($line.Trim() -match "###\s(?<sectionName>.*)")
+          {
+            $sectionName = $matches["sectionName"]
+            $changeLogEntry.ReleaseContent += $line
+            continue
+          }
+
+          if (-not [System.String]::IsNullOrEmpty($line))
```

Comment:
Do we really want to strip out empty lines? That would eliminate any spacing that folks put inside of a given section.

Reply:
If you are just trying to eliminate the lines at the top and bottom of a section we can do that after the fact.

Reply:
I was truing to not and empty line when there is no entry in the section.